definitions/features/qpid_queues.rb

```diff
@@ -0,0 +1,47 @@
+class Features::QpidQueues < ForemanMaintain::Feature
```

Comment:
I think we could generalize this to `qpid` feature, and collect mutliple functionalities around this feature in the future